src/migrations/dapps/1718910535585_create-trades-and-trade-assets-table.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const TRADES_TABLE = 'trades'
```

Comment:
Are we going to use schemas for the data? For example, the `marketplace` schema in this case.